src/error.rs

```diff
@@ -12,4 +12,10 @@ pub enum Error {
         "Found `{0}` but value MUST only contain numbers, letters, and the characters ., /, and -."
     )]
     InvalidStackId(String),
+    #[error("could not serialize into TOML")]
+    TomlSerError(#[from] toml::ser::Error),
+    #[error("could not deserialize from TOML")]
+    TomlDeError(#[from] toml::de::Error),
+    #[error("I/O Error")]
+    IoError(#[from] std::io::Error),
```

Comment:
Will this make the `Display` of that `io::Error` appear somewhere? If not, that would complicate debugging.